axum-extra/src/extract/multipart.rs

```diff
@@ -246,6 +247,58 @@ impl MultipartError {
     fn from_multer(multer: multer::Error) -> Self {
         Self { source: multer }
     }
+
+    /// Get the response body text used for this rejection.
+    pub fn body_text(&self) -> String {
+        self.source.to_string()
+    }
+
+    /// Get the status code used for this rejection.
+    pub fn status(&self) -> http::StatusCode {
+        status_code_from_multer_error(&self.source)
+    }
+}
+
+fn status_code_from_multer_error(err: &multer::Error) -> StatusCode {
+    match err {
+        multer::Error::UnknownField { .. }
+        | multer::Error::IncompleteFieldData { .. }
+        | multer::Error::IncompleteHeaders
+        | multer::Error::ReadHeaderFailed(..)
```

Comment:
I'm not sure if this `multer::Error::ReadHeaderFailed` is always thrown by the client side. How do you think? Is it OK to response as `400`?

Reply:
Feels like if the headers cannot be parsed then that's a client issue. The inner error is [`httparse::Error`](https://docs.rs/httparse/1.8.0/httparse/enum.Error.html)